docs/GeneralSetup.md

```diff
@@ -153,21 +153,21 @@ These steps can be done in the Terminal/CLI or with PowerShell.
     dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --prerelease
     ```
 
-    **JavaScript, TypeScript, Python, Powershell:** Update the `host.json` file to the preview extension bundle.
+    **JavaScript, TypeScript, Python, Powershell:** Update the `host.json` file to the v4 version of the extension bundle.
 
     ```json
     "extensionBundle": {
-        "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+        "id": "Microsoft.Azure.Functions.ExtensionBundle",
         "version": "[4.*, 5.0.0)"
```

Comment:
need to update the version as well for Microsoft.Azure.Functions.ExtensionBundle to [3.*, 4.0.0)

Reply:
Is there a reason we'd want to be in the v3 of the bundle and not v4?

Reply:
oops I thought the latest ExtensionBundle is V3 (when I created a function through func init it has the old default values) :| Never mind then 👍